template.rb

```diff
@@ -84,6 +96,45 @@
 end
 CODE
 
+if yes?("Are you hosting Shipit on Heroku? (y/n)")
+  inject_into_file "Procfile", " -p $PORT", after: "web: bundle exec rails s"
+  inject_into_file "Gemfile", "ruby '2.2.3'", after: "source 'https://rubygems.org'\n"
+
+  gsub_file 'Gemfile', "# Use sqlite3 as the database for Active Record", ''
+  gsub_file 'Gemfile', "gem 'sqlite3'", ''
+  gem_group :production do
+    gem 'pg'
+    gem 'rails_12factor'
+  end
+  gem_group :development, :test do
+    gem 'sqlite3'
+  end
+
+  remove_file 'config/database.yml'
+  create_file 'config/database.yml', <<-EOF
```

Comment:
> I was not sure if people typically use sqlite for the production db

For webservers, I doubt it's a sane default.

Can you use the correct indentation for the code along with `.strip_heredoc` please?